src/sunmatrix/onemkldense/sunmatrix_onemkldense.cpp

```diff
@@ -591,27 +591,27 @@ int SUNMatMatvec_OneMklDense(SUNMatrix A, N_Vector x, N_Vector y)
                             xdata, 1, ZERO, ydata, 1);
   }
 
-  return SUNMAT_SUCCESS;
+  return SUN_SUCCESS;
 }
 
-int SUNMatSpace_OneMklDense(SUNMatrix A, long int* lenrw, long int* leniw)
+SUNErrCode SUNMatSpace_OneMklDense(SUNMatrix A, long int* lenrw, long int* leniw)
 {
   if (!A)
   {
     SUNDIALS_DEBUG_ERROR("Input matrix is NULL\n");
-    return SUNMAT_ILL_INPUT;
+    return SUN_ERR_ARG_INCOMPATIBLE;
   }
 
   if (SUNMatGetID(A) != SUNMATRIX_ONEMKLDENSE)
   {
     SUNDIALS_DEBUG_ERROR("Invalid matrix ID\n");
-    return SUNMAT_ILL_INPUT;
+    return SUN_ERR_ARG_INCOMPATIBLE;
```

Comment:
Update to `SUNAssert` and `SUN_ERR_ARG_WRONGTYPE`

Reply:
```suggestion
    return SUN_ERR_ARG_WRONGTYPE;